v2.1/reference/cni-plugin/configuration.md

```diff
@@ -17,6 +17,19 @@ A minimal configuration file that uses Calico for networking and IPAM looks like
 }
 ```
 
+If the `calico-node` container on a node registered with a `NODENAME` other than the node hostname, the CNI plugin on this node must be configured with the same `nodename`:
+
+```json
+{
+    "name": "any_name",
+    "nodename": "the-calico-node-NODENAME",
```

Comment:
Could we change `"the-calico-node-NODENAME"` to `"<NODENAME>"` as i think we tend to use the angles for substitutions in our docs IIRC.